From 761448f2d1bda5c06c6377d468867c2dc8beb4f3 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Tue, 30 Aug 2022 18:16:16 +0200
Subject: [PATCH] [CMake] Search for Python & Boost::Python

as eigenpy doesn't need the interpreter, rely on it is not enough
---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- CMakeLists.txt.orig	2022-08-30 14:36:10.000000000 +0200
+++ CMakeLists.txt	2022-08-30 22:28:12.547328989 +0200
@@ -28,9 +28,9 @@
 check_minimal_cxx_standard(14 ENFORCE)
 
 # Project dependencies
+set(PYTHON_COMPONENTS Interpreter Development NumPy)
 add_project_dependency(dynamic-graph 4.4.0 REQUIRED)
 add_project_dependency(eigenpy 2.7.10 REQUIRED)
-include(cmake/python.cmake) # TODO: overwriting eigenpy/python.cmake
 if(BUILD_TESTING)
   find_package(Boost REQUIRED COMPONENTS unit_test_framework)
 endif(BUILD_TESTING)
